modules/utxo-lib/test/integration_local_rpc/parseDashEvo.ts

```diff
@@ -26,8 +31,13 @@ export async function readDashEvoTransactions(): Promise<RpcTransaction[]> {
 
 describe('Dash', function () {
   const network = networks.dash;
-  it(`parses Evolution (EVO) special transactions`, async function () {
-    const txs = await readDashEvoTransactions();
+  let txs: RpcTransaction[];
+
+  before('read fixtures', async function () {
+    txs = await readDashEvoTransactions();
+  });
+
+  it(`parses Evolution (EVO) special transactions`, function () {
```

Comment:
I wonder what the `EVO` terminology used here is compared to the `proReg` terminology used in HSM. Don't understand Dash. Should we be helping future engineers by using both terms in both places or only one in both?